### ISSUE TITLE
error with task shell echo with parentheses marks

### ISSUE THREAD

**Issue by sibo** [nwchemgit/nwchem#859]
**Describe the bug**
The following line in an input file results in NWChem 6.8 failing:

`task shell "echo @starting single-point energy (anything here)"`

If the parenthetical portion is removed, the input file works fine.

`task shell "echo @starting single-point energy"`

I have also reproduced this issue with other parenthetical task shell echo statements. I have confirmed this issue exists even if the task shell statement is the only line in the whole input file.

**Describe settings used**
Running on NWChem 6.8. Apologies if this issue was reported and fixed in 7.2

**Attach log files**
```
                                NWChem Input Module
                                -------------------


 ------------------------------------------------------------------------
 shell failed                                                                                       0
 ------------------------------------------------------------------------
 ------------------------------------------------------------------------
  current input line :
     1: task shell "echo @starting single-point energy()"
 ------------------------------------------------------------------------
 ------------------------------------------------------------------------
 This type of error is most commonly associated with calculations not reaching convergence criteria
 ------------------------------------------------------------------------
 For more information see the NWChem manual at http://www.nwchem-sw.org/index.php/NWChem_Documentation


 For further details see manual section:                                                                                                                                                                                                                          

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 22064 RUNNING AT hrcnode089
=   EXIT CODE: 255
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================
```

**Comment by hjjvandam** [comment]
Hi,
Could this failure be the result of the shell command? I.e. if I copy and paste the following into bash or sh I get:


                sh-3.2$ echo @starting single-point energy (anything here)

sh: syntax error near unexpected token `('

I don’t think that you can expect NWChem to be able to run a shell command that the shell itself cannot. Can you rephrase the command without the round brackets? That might solve your problem.
Huub
-----------------------------------------------------------------------------------------------------
Hubertus van Dam, 631-344-6020, ***@***.******@***.***>
Brookhaven National Laboratory


From: sibo ***@***.***>
Date: Friday, September 8, 2023 at 9:14 AM
To: nwchemgit/nwchem ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nwchemgit/nwchem] error with task shell echo with parentheses marks (Issue #859)

Describe the bug
The following line in an input file results in NWChem 6.8 failing:

task shell "echo @starting single-point energy (anything here)"

If the parenthetical portion is removed, the input file works fine.

task shell "echo @starting single-point energy"

I have also reproduced this issue with other parenthetical task shell echo statements. I have confirmed this issue exists even if the task shell statement is the only line in the whole input file.

Describe settings used
Running on NWChem 6.8. Apologies if this issue was reported and fixed in 7.2

Attach log files

                                NWChem Input Module

                                -------------------





 ------------------------------------------------------------------------

 shell failed                                                                                       0

 ------------------------------------------------------------------------

 ------------------------------------------------------------------------

  current input line :

     1: task shell "echo @starting single-point energy()"

 ------------------------------------------------------------------------

 ------------------------------------------------------------------------

 This type of error is most commonly associated with calculations not reaching convergence criteria

 ------------------------------------------------------------------------

 For more information see the NWChem manual at http://www.nwchem-sw.org/index.php/NWChem_Documentation<https://urldefense.com/v3/__http:/www.nwchem-sw.org/index.php/NWChem_Documentation__;!!P4SdNyxKAPE!DGpwC6koht43Uk1ABjAZqVmVQVksllRk_RSewITC1ZWti4QsZXLROo3lOGblCFF8Bl7t-otkvIvnkXsAtRe1rYU9$>





 For further details see manual section:



===================================================================================

=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES

=   PID 22064 RUNNING AT hrcnode089

=   EXIT CODE: 255

=   CLEANING UP REMAINING PROCESSES

=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES

===================================================================================

**Comment by sibo** [comment]
Huub, Thank you for the clarification!
